common/src/main/java/com/wynntils/screens/guides/charm/WynntilsCharmGuideScreen.java

```diff
@@ -30,7 +31,9 @@ public final class WynntilsCharmGuideScreen
     private List<GuideCharmItemStack> allCharmItems = List.of();
 
     private WynntilsCharmGuideScreen() {
-        super(Component.translatable("screens.wynntils.wynntilsGuides.charmGuide.name"));
+        super(
```

Comment:
This formatting looks really odd! Is this the doing of Spotless?

Reply:
It is. There are a few instances, where it is not perfect...

https://github.com/Wynntils/Artemis/blob/main/common/src/main/java/com/wynntils/utils/render/buffered/BufferedFontRenderer.java#L68

Reply:
*sigh* :-(